### PR TITLE
History information presenter

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
First work. Things to improve later on (on demand when new requirements come in): Permissions and displaying of the user.

### CODE REVIEW

File: openslides_backend/presenter/get_history_information.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    Return all history information for one fqid.
+    """
+
+    schema = get_history_information_schema
+
+    def get_result(self) -> Any:
+        if not has_organization_management_level(
+            self.datastore, self.user_id, OrganizationManagementLevel.SUPERADMIN
+        ):
+            raise PermissionDenied("You are not a superadmin")
```
Comment (jsangmeister) [inline openslides_backend/presenter/get_history_information.py:39]: This reduces the feature set from OS3 since only we are superadmins, so the organization admins can no longer look at the history or restore it. Is this intended?

File: tests/system/presenter/base.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ def get_application(self) -> WSGIApplication:
 
     def request(
         self, presenter: str, data: Optional[Dict[str, Any]] = None
-    ) -> Tuple[int, Dict[str, Any]]:
+    ) -> Tuple[int, Any]:
```
Comment (jsangmeister) [inline tests/system/presenter/base.py:14]: Why did you change this?

Comment (FinnStutzenstein) [reply tests/system/presenter/base.py:14]: The presenter returns a list instead of an object.

File: openslides_backend/presenter/get_history_information.py (hunk 1 above)
Comment (FinnStutzenstein) [reply openslides_backend/presenter/get_history_information.py:39]: This is right.There is the meeting perm `can_see_history` but I'm unsure if this is a good permission:

IMO it should be coupled to the admin of the meeting. In OS3 you can see the history list with this permission, but can not enter history mode (you need the OS3 superadmin for it). I think it was the case since we could not restrict the histoy data, so the superadmin was needed. We need a permission concept for the history list and retrieving history data. Until then, I'll leave it for the superadmin. I think how it work has to be discussed with Emanuel.
